front/src/reducers/osrdconf2/simulationConf/index.ts

```diff
@@ -0,0 +1,31 @@
+import { createSlice, ListenerMiddlewareInstance } from '@reduxjs/toolkit';
+import { OsrdConfState } from 'applications/operationalStudies/consts';
+import {
+  defaultCommonConf,
+  buildCommonOsrdConfReducers,
+  addCommonOsrdConfMatchers,
+  registerUpdateInfraIDListener,
+} from '../common';
+
+export const simulationConfInitialState = defaultCommonConf;
+
+export const simulationConfSlice = createSlice({
+  name: 'simulationconf',
+  initialState: simulationConfInitialState,
+  reducers: { ...buildCommonOsrdConfReducers<OsrdConfState>() },
```

Comment:
Can you harmonize the names ? defaultCommonConf, buildCommonConfReducers, buildCommonConfSelectors, testCommonConfReducers 
```suggestion
  reducers: { ...buildCommonConfReducers<OsrdConfState>() },
```